pyaedt/modules/SolveSetup.py

```diff
@@ -538,6 +538,112 @@ def add_eddy_current_sweep(self, range_type="LinearStep", start=0.1, end=100, co
         self.auto_update = legacy_update
         return True
 
+    @pyaedt_function_handler()
+    def enable_adaptive_setup_single(self, freq=None, max_passes=None, max_delta_s=None):
+        """Enable Hfss Single Frequency Setup:
+
+        Parameters
+        ----------
+        freq : float, str
+            Frequency at which set the adaptive convergence. It can be float (GHz) or str.
+        max_passes : int
+            Maximum number of adaptive passes.
+        max_delta_s : float
+            Delta S Convergence criteria.
+
+        Returns
+        -------
+        bool
+        """
+        if self.setuptype != 1:
+            self._app.logger.error("Method applies only to Hfss Driven Solutions.")
+            return False
+        self.auto_update = False
+        self.props["SolveType"] = "Single"
+        if isinstance(freq, (int, float)):
+            freq = "{}GHz".format(freq)
+        if freq:
+            self.props["Frequency"] = freq
+        if max_passes:
+            self.props["MaximumPasses"] = max_passes
+        if max_delta_s:
+            self.props["MaxDeltaS"] = max_delta_s
+        self.auto_update = True
+        return self.update()
+
+    @pyaedt_function_handler()
+    def enable_adaptive_setup_broadband(self, low_frequency, high_frquency, max_passes=6, max_delta_s=0.02):
+        """Enable Hfss Broadband Setup.
+
+        Parameters
+        ----------
+        low_frequency : float, str
+            Lower Frequency at which set the adaptive convergence. It can be float (GHz) or str.
+        high_frquency : float, str
+            Lower Frequency at which set the adaptive convergence. It can be float (GHz) or str.
+        max_passes : int
+            Maximum number of adaptive passes.
+        max_delta_s : float
+            Delta S Convergence criteria.
+
+        Returns
+        -------
+        bool
+        """
+        if self.setuptype != 1:
+            self._app.logger.error("Method applies only to Hfss Driven Solutions.")
+            return False
+        self.auto_update = False
+        self.props["SolveType"] = "BroadBand"
+        for el in list(self.props["MultipleAdaptiveFreqsSetup"].keys()):
+            del self.props["MultipleAdaptiveFreqsSetup"][el]
+        if isinstance(low_frequency, (int, float)):
+            low_frequency = "{}GHz".format(low_frequency)
+        if isinstance(high_frquency, (int, float)):
+            high_frquency = "{}GHz".format(high_frquency)
```

Comment:
high_frequency line 603 and 605

Reply:
is it still "high_frquency" in version 0.11? when i use "high_frequency", hfss raise error like this. when i switch it to "high_frquency", it worked
![image](https://github.com/user-attachments/assets/4a7cf1f0-1d6e-4bf3-9524-c38567263204)